packages/mds-policy-author/api.ts

```diff
@@ -89,17 +86,24 @@ function api(app: express.Express): express.Express {
         return res.status(200).send({ version: res.locals.version, data: { policy } })
       } catch (error) {
         logger.error('failed to publish policy', error.stack)
-        if (error instanceof AlreadyPublishedError) {
-          return res.status(409).send({ error })
-        }
-        if (error instanceof NotFoundError) {
-          return res.status(404).send({ error })
-        }
-        if (error instanceof DependencyMissingError) {
-          return res.status(424).send({ error })
+        logger.error('typeof', error.constructor.name)
+        switch (error.constructor.name) {
```

Comment:
If only TS had a nice way of switching on types, alas

Reply:
yeah it made me sad too